### PR TITLE
Fix peer dependency vulnerable packages

### PR DESCRIPTION
**Pull request by Ben0189**
~Temporary override peer dependency packages ws and micromatch~

~Reason: To fix vulnerabilities, as the parent packages dependency used in tinacms haven't get the latest upgrade of both packages~

Update some packages

### CODE REVIEW

File: .changeset/brave-eagles-live.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+'next-tinacms-s3': patch
+'tinacms-authjs': patch
+'tinacms-clerk': patch
+'e2e-next': patch
+'@tinacms/app': patch
+'@tinacms/cli': patch
+'@tinacms/mdx': patch
+'next-2024': patch
+'tinacms': patch
+'starter-empty': patch
```
Comment (JackDevAU) [inline .changeset/brave-eagles-live.md:23]:
```suggestion
'tinacms': patch
```

File: package.json [hunk 2]
```diff
@@ -48,5 +48,13 @@
 		"typescript": "^5.5.4",
 		"vite": "^4.3.9"
 	},
+	"resolutions": {
+		"micromatch": "4.0.8",
+		"ws": "8.17.1"
+	  },
+	"packageComments":{
+		"micromatch": "Overriding the version of micromatch to 4.0.8 to fix a vulnerability issue, Remove the override once the parent packages @changesets/cli and @typescript-eslint/parser are updated to a version that use latest micromatch.",
+		"ws": "Overriding the version of ws to 8.17.1 to fix a vulnerability issue. Remove the override once the parent package tailwind is updated to a version that use latest ws."
```
Comment (JackDevAU) [inline package.json:57]: I think we should just make an issue/PR on there side to update these 🙏 
This way we don't add this Tech Debt to our repo (which already has quite a lot ;(

Comment (Ben0189) [reply package.json]: https://github.com/changesets/changesets/issues/1448

File: .changeset/brave-eagles-live.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'tinacms': patch
+---
+
+fix vulnerability packages
```
Comment (JackDevAU) [inline .changeset/brave-eagles-live.md:5]:
```suggestion
Updates`puppeteer` & `@changesets/cli`  packages
```

**Review by JackDevAU — APPROVED** [review]

**Pull request merged**